### DISCUSSION TITLE
kindle-agent这个程序是不是没开源啊

### DISCUSSION THREAD

**Discussion by hugbgithub** [harmonycloud/kindling#652] [agent probe]
编译出来只有kindle-collector，kindle-agent好像只能docker 部署

**Comment by litaixun** [comment]
kindling-collector 包含了 kindling-agent，你说的 kindling-agent 是底层的 eBPF probe 吗？probe 目前是从仓库下载 .o 文件直接打进镜像使用脚本加载使用的，底层是基于另一个项目 github.com/falcosecurity/libs 二次开发的，源码是这个 github.com/KindlingProject/agent-libs 项目。
